### PR TITLE
OSSMDOC-516: Draft of service mesh z-stream release notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
- Aligned team: Dev Tools
- For branches: 4.6, 4.7, 4.8, 4.9, 4.10, 4.11
- Jira: https://issues.redhat.com/browse/OSSMDO-516
- Direct link to **2.1.2 and 2.0.9** doc preview: https://deploy-preview-42275--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html
- Direct link to **1.1.18** doc preview: https://deploy-preview-42275--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v1x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-1-1-18
- SME review: @longmuir
- QE review: @yxun, @tsze-redhat, @mattmahoneyrh 
- Peer review: @JStickler

### CODE REVIEW

**Comment by longmuir** [comment]
LGTM

File: modules/ossm-rn-new-features.adoc [hunk 1]
```diff
@@ -19,7 +19,11 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
-== Component versions included in {SMProductName} version {SMProductVersion}
+== New features {SMProductName} 2.1.2
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
```
Comment (jwendell) [inline modules/ossm-rn-new-features.adoc:26]: `{SMProductVersion}` is shown is the preview as `2.1.1`. I'm guessing this is going to be fixed automatically, right?

File: modules/ossm-rn-new-features.adoc [hunk 2]
```diff
@@ -43,6 +47,24 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 
 This release also adds the ability to disable the automatic creation of network policies.
 
+=== Component versions included in {ProductName} version 2.1.1
+
+|===
+|Component |Version
+
+|Istio
+|1.9.6
```
Comment (jwendell) [inline modules/ossm-rn-new-features.adoc:56]: 1.9.9

File: modules/ossm-rn-new-features.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:26]: Thanks for pointing this out. We started using a new attributes file, which has now been updated. This change has been made.

File: modules/ossm-rn-new-features.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: This change was made

File: modules/ossm-rn-new-features.adoc [hunk 3]
```diff
@@ -141,6 +179,28 @@ Kiali 1.36 includes the following features and enhancements:
...
+=== Component versions included in {SMProductName} version 2.0.9
+
+|===
+|Component |Version
+
+|Istio
+|1.9.6
+
+|Envoy Proxy
+|1.17.1
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: This change was made

File: modules/ossm-rn-new-features.adoc [hunk 4]
```diff
@@ -141,6 +179,28 @@ Kiali 1.36 includes the following features and enhancements:
...
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions included in {SMProductName} version 2.0.9
+
+|===
+|Component |Version
+
+|Istio
+|1.9.6
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: This change was made.

**Comment by rh-tokeefe** [comment]
> Under this section https://deploy-preview-42275--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-1-2
> 
> I see https://deploy-preview-42275--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#component-versions-included-in-red-hat-openshift-service-mesh-version-2-1-1
> 
> Is there supposed to be a section for 2.1.2 similar to the one above for 2.1.1?

We're using a new attributes file that populates the version variable with a number. The file has now been updated, so the 2.1.2 version appears in the heading.

**Review by jshaughn — APPROVED** [review]
kiali versioning looks good.

File: modules/ossm-rn-new-features.adoc [hunk 5]
```diff
@@ -43,6 +47,24 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
...
+|Component |Version
+
+|Istio
+|1.9.9
+
+|Envoy Proxy
+|1.17.1
+
+|Jaeger
+|1.24.1
```
Comment (JStickler) [inline modules/ossm-rn-new-features.adoc:62]: The latest Jaeger version is 1.30.1

**Review by JStickler — COMMENTED** [review]
Once you update the Jaeger version and remove the [WIP] this should be ready to go.  LGTM other than those two small updates.

**Review by mattmahoneyrh — COMMENTED** [review]
Feedback added.

File: modules/ossm-rn-new-features.adoc [hunk 6]
```diff
@@ -81,9 +103,25 @@ spec:
 
 == New features and enhancements {SMProductName} 2.1
 
-This release of {SMProductName} adds support for Istio 1.9.8, Envoy Proxy 1.17.1, Jaeger 1.24.1, and Kiali 1.36.5 on {product-title} 4.6 EUS, 4.7, 4.8, and 4.9.
+This release of {SMProductName} adds support for Istio 1.9.8, Envoy Proxy 1.17.1, Jaeger 1.24.1, and Kiali 1.36.5 on {product-title} 4.6 EUS, 4.7, 4.8, 4.9, along with new features and enhancements.
+
```
[the remark this answers is not in the document]
Comment (jwendell) [reply modules/ossm-rn-new-features.adoc:107]: Istio 1.12 == OSSM 2.2 which is WiP and is not part of this PR.

File: modules/ossm-rn-new-features.adoc [hunk 7]
```diff
@@ -43,6 +47,24 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 
 This release also adds the ability to disable the automatic creation of network policies.
 
+=== Component versions included in {ProductName} version 2.1.1
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:50]: Matt and I looked at this together. We confirmed his comment was placed in the 2.1.1 section of the release notes.The existing content is correct for 2.1.1. 

We also looked together at Release Readiness Doc and the 2.1.2 section of the release notes. We have ensured the Service Mesh version is correct.

File: modules/ossm-rn-new-features.adoc [hunk 8]
```diff
@@ -43,6 +47,24 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
...
+|1.9.9
+
+|Envoy Proxy
+|1.17.1
+
+|Jaeger
+|1.24.1
+
+|Kiali
+|1.36.7
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:65]: Matt and I looked at this together. We confirmed his comment was placed in the 2.1.1 section of the release notes.The existing content is correct for 2.1.1. 

We also looked together at Release Readiness Doc and the 2.1.2 section of the release notes. We have ensured the component version is correct.

File: modules/ossm-rn-new-features.adoc (hunk 6 above)
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:107]: Matt and I looked at this together. We confirmed his comment was placed in the 2.1.1 section of the release notes.The existing content is correct for 2.1.1. 

We also looked together at Release Readiness Doc and the 2.1.2 section of the release notes. We have ensured the component version is correct.

File: modules/ossm-rn-new-features.adoc [hunk 9]
```diff
@@ -81,9 +103,25 @@ spec:
...
-In addition, this release has the following new features and enhancements:
+|Envoy Proxy
+|1.17.1
+
+|Jaeger
+|1.24.1
+
+|Kiali
+|1.36.5
+|===
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:124]: Matt and I looked at this together. We confirmed his comment was placed in the 2.1.1 section of the release notes.The existing content is correct for 2.1.1. 

We also looked together at Release Readiness Doc and the 2.1.2 section of the release notes. We have ensured the component version is correct.

File: modules/ossm-rn-new-features.adoc (hunk 5 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:62]: Updated the Jaeger version in the 2.1.2 section of rel notes.

File: modules/ossm-rn-new-features.adoc [hunk 10]
```diff
@@ -141,6 +179,28 @@ Kiali 1.36 includes the following features and enhancements:
...
+
+|Envoy Proxy
+|1.14.5
+
+|Jaeger
+|1.24.1
+
+|Kiali
+|1.24.11
+|===
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:202]: Matt and I reviewed and resolved this. Done.

**Pull request merged**